app/views/users/confirmations/show.html.erb

```diff
@@ -8,9 +8,14 @@
       <%= form_for(resource, as: resource_name, url: users_confirmations_path, html: { method: :put }) do |f| %>
         <%= f.hidden_field :confirmation_token, value: params[:confirmation_token] || params[:user][:confirmation_token] %>
 
+        <div class="govuk-form-group <%= "govuk-form-group--error" if resource&.errors&.any? %>">
```

Comment:
This is going to set the red error flag on the left hand side of both the password and name fields when only one has an error. 

The guidance is that you add the red flag only to the field in question: https://design-system.service.gov.uk/components/error-message/

We probably need to do something like use the `resource.errors` to identify the field by `id` and add the error class.